packages/create/src/helpers.js

```diff
@@ -1,4 +1,13 @@
 import qoa from 'qoa';
+import commandLineArgs from 'command-line-args';
+
+const optionDefinitions = [
+  { name: 'tag-name', type: String, defaultValue: '' },
+  { name: 'no-npm', type: Boolean, defaultValue: false },
+  { name: 'no-scaffold', type: Boolean, defaultValue: false },
+];
```

Comment:
come to think of it I guess having it like this might be better
```
npm init @open-wc --tag-name foo-bar --npm no --scaffold yes
```

e.g. we may want to force in "both" direction...

Reply:
Alright, so:

`--no-npm` is just a boolean now. Its false by default, and when specified it wont npm install
`--no-scaffold` is a string. If not specified, will prompt the user. If specified as `--no-scaffold=true`, it will not prompt and not scaffold, if specified as `--no-scaffold=false` it will not prompt but it will scaffold.

Reply:
we changed approach